entry-points/spin-entry-point.js

```diff
@@ -30,14 +38,14 @@ class SpinderUi extends Component { {
         <View style={{highlight: 'love'}}>
           {profileImage}
           <Text style={{text: 'excite'}}>
-            {`LOOK! IT'S ${firstName} ${lastName}. Could be the one!`}
+            {`LOOK! IT'S ${prefferedName}. Could be the one!`}
```

Comment:
looks like we aren't defining `preferredName` (misspelling on the variable too).